mbl/urls.py

```diff
@@ -33,6 +33,9 @@
     url(r'^person/(?P<person_id>[0-9]+)/split/$', views.split_person, name="split-person"),
     url(r'^person/merge/$', views.merge_people, name="merge-people"),
     url(r'^person/$', views.person, name="person_list"),
+    url(r'^person/(?P<person_id>[0-9]+)/investigator/add/$',views.add_investigator_record,name='add-investigator'),
+    url(r'^person/(?P<person_id>[0-9]+)/investigator/(?P<research_id>[0-9]+)/edit/$',views.edit_investigator_record,name='edit-investigator'),
+    url(r'^person/(?P<person_id>[0-9]+)/investigator/delete/$', views.delete_investigator_record, name='delete-investigator'),
```

Comment:
same as the edit, it should be ```investigator/[id]/delete```

Reply:
This line of code <b>extends "browser/base.html</b> is used to form the structure of the html page. So that is why I have not removed it.

I have added <b>investigator/id/delete</b> in the urls.py file.

Reply:
my comment for person.html is not about the line of code but the file permission. please read my original comment.

Reply:
I have changed the permission from 755 to 644 of <b>person.html</b>